acceptance/chaos_test.go

```diff
@@ -225,6 +269,16 @@ func TestChaos(t *testing.T) {
 		}
 	}
 
+	// Verify accounts.
+	db = makePGClient(t, c.PGUrl(0))
```

Comment:
Is there a reason you're create another `db` here? If so, it isn't obvious and should be commented.